.gitignore

```diff
@@ -47,3 +47,4 @@ test-results/
 
 # Localization Resources
 /server/locales/**/*.yml
+yarn.lock
```

Comment:
I would leave this file non-ignored, as you shouldn't add lock files to .gitignore.